label_studio/tasks/serializers.py

```diff
@@ -711,13 +658,19 @@ def get_unique_lock_id(self, task):
             return lock.unique_id
 
     def get_predictions(self, task):
-        project = task.project
-        if not project.show_collab_predictions:
-            return []
-        else:
-            for ml_backend in project.ml_backends.all():
-                ml_backend.predict_tasks([task])
-            return super().get_predictions(task)
+        """ """
+        predictions = task.get_predictions_for_prelabeling()
+        return PredictionSerializer(predictions, many=True, read_only=True, default=[], context=self.context).data
+
+    # def get_predictions(self, task):
+    #     project = task.project
+
+    #     if not project.show_collab_predictions:
+    #         return []
+    # else:
+    #     # for ml_backend in project.ml_backends.all():
+    #     #     ml_backend.predict_tasks([task])
+    #     return super().get_predictions(task)
```

Comment:
Need to make sure we didn't break anything with this change, and remove commented out code